usaspending_api/api_docs/api_documentation/download/advanced_search_transaction_download.md

```diff
@@ -0,0 +1,58 @@
+## [Advanced Search Transaction Download](#Advanced_Search_Transaction_Download)
+**Route:** `/api/v2/download/transactions/`
+
+**Method:** `POST`
+
+This route sends a request to the backend to begin generating a zipfile of transaction data in CSV form for download.
+
+### Request example
+
+```
+{
+    "filters": {},
+    "columns": [
+        "award_type",
+        "awarding_agency_code"
+    ]
+}
+```
+
+### Request Parameters Description
+
+* `filters` - *required* - a standard Search v2 JSON filter object
```

Comment:
It would be helpful to link to the Search v2 JSON filter documentation

Reply:
Same in `download/advanced_search_award_download`

Reply:
noted

Reply:
added